<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OgrenciWidget</class>
 <widget class="QWidget" name="OgrenciWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="ogranciEkleButton">
     <property name="text">
      <string>Ogrenci Ekle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ogrenciSilButton">
     <property name="text">
      <string>Ogrenci Sil</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="ogrenciListView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
